examples/example_pipeline.ipynb

```diff
@@ -58,7 +58,8 @@
     "        'scale_args': {\n",
     "            \"method\": \"general\",\n",
     "            \"scaler_name\": \"MinMaxScaler\"\n",
-    "        }\n",
+    "        },\n",
+    "        'allow_modified' : False\n",
```

Comment:
This is the new parameter exposed to the user side

Reply:
After local testing, with allow_modified = False, no _foundation and _scaled columns are shown in the post-processed dataframes.